src/Microsoft.Extensions.Logging.AzureWebAppDiagnostics/Internal/FileLoggerProvider.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Serilog;
+using Serilog.Formatting.Display;
+using Serilog.Sinks.RollingFile;
+
+namespace Microsoft.Extensions.Logging.AzureWebAppDiagnostics.Internal
+{
+    /// <summary>
+    /// A file logger for Azure WebApp.
+    /// </summary>
+    public class FileLoggerProvider : SerilogLoggerProvider
```

Comment:
Minor thing, but instead of subclassing `SerilogLoggerProvider`, using an instance variable and delegating to it will keep `SLP` off of the public API surface of this type (in case a different implementation is desired in future - changing base classes is a breaking change).